src/main/java/jenkins/plugins/slack/workflow/SlackResponse.java

```diff
@@ -42,4 +49,17 @@ public String getThreadId() {
             return null;
         }
     }
+
+    /**
+     * Add an emoji reaction to the message that this `SlackResponse` points to.
+     *
+     * @param emojiName - name of the emoji (no colons), e.g. `thumbsup`
+     *
+     * @return boolean indicating whether the API request succeeded
+     */
+    @Whitelisted
+    public boolean addReaction(String emojiName) {
+        if (slackService == null) return false;
```

Comment:
please add braces around if statements
```suggestion
        if (slackService == null) {
          return false;
        }
```